usr/lib/common/loadsave.c

```diff
@@ -1081,10 +1146,9 @@ CK_RV load_masterkey_user_old(STDLL_TokData_t *tokdata)
     // this file gets created on C_InitToken so we can assume that it always
     // exists
     //
-    sprintf(fname, "%s/MK_USER", tokdata->data_store);
-    fp = fopen(fname, "r");
+    fp = open_token_data_store_path(fname, sizeof(fname), tokdata, "MK_USER",
+                                    "r");
```

Comment:
same here - it is much more better readable to have the "r"); moved back to were it belongs.

Reply:
Agreed and changed.